cogs/help.py

```diff
@@ -38,27 +52,85 @@ async def create_view(self):
             self.bot.add_view(HelpView())
 
     @commands.Cog.listener()
-    async def on_message(self, message):
+    async def on_message(self, message: Message):
         if message.channel.id == self.help_channel and message.type == MessageType.thread_created:
             await message.delete(delay=5)
 
-    @commands.command()
+    @commands.command(hidden=True)
     @commands.is_owner()
-    async def help_menu(self, ctx):
+    async def help_menu(self, ctx: commands.Context):
         await ctx.send("Click a button to create a help thread!", view=HelpView())
 
-    @commands.command()
-    async def close(self, ctx):
+    @commands.command(name="Close", aliases=['cs'])
```

Comment:
Why is commands uppercase now

Reply:
thats how the help command works. i mean if you want it to be lowercase then lower case ig

Reply:
That would require you to use =Close...?

Reply:
wait you havent set it to `case_insenstive=True`??

Reply:
no

Reply:
this was never resolved but has been marked as so